mits/models.py

```diff
@@ -3,34 +3,64 @@
 
 @Session.model_mixin
 class SessionMixin:
-    def logged_in_mits_team(self):
+    def log_in_as_mits_team(self, team_id, redirect_to='../mits_applications/index'):
         try:
-            return self.mits_team(cherrypy.session['mits_team_id'])
+            team = self.mits_team(team_id)
+            duplicate_teams = []
+            while team.duplicate_of:
+                duplicate_teams.append(team.id)
+                team = self.mits_team(team.duplicate_of)
+                assert team.id not in duplicate_teams, 'circular reference in duplicate_of: {}'.format(duplicate_teams)
```

Comment:
I absolutely love this idea! I am super happy anytime we can support outdated URLs :smile:

Might want to consider using a `set()` for `duplicate_teams`, but I can't imagine there would ever be enough duplicates for the performance to matter.